packages/browser/test/index.test.ts

```diff
@@ -4,38 +4,55 @@ const DONE = window.XMLHttpRequest.DONE
 
```

Comment:
I reworked the mocking in this test suite to get a callback on the xhr send where I could call done. The existing mocking wasn't able to handle the awaiting of the getIntegrity promise.